### PR TITLE
Update: Data source parameter table

### PR DESCRIPTION
**Pull request by osisoft-csleasman**
These updates are to bring BACnet doc into consistency with OPCUA doc based on feedback from the SPM.

### CODE REVIEW

**Review by osisoft-dzhang — APPROVED** [review]

**Review by osisoft-izinman — COMMENTED** [review]
Couple places where clarity could be improved

File: V1/Configuration/pi-adapter-for-bacnet-data-source-configuration.md [hunk 1]
```diff
@@ -55,8 +55,8 @@ The following parameters are available to configure a BACnet data source:
...
-| **DefaultStreamIdPattern** | Optional | `string` | Specifies the default stream ID pattern to use. Possible parameters: {DeviceIPAddress}, {DeviceId}, {ObjectType}, {ObjectId}, and {PropertyIdentifier}. An empty or `null` value will result in {DeviceId}.{ObjectType}{ObjectId}.{PropertyIdentifier}.|
+| **StreamIdPrefix** | Optional | `string` | Specifies what prefix is used for stream IDs. The naming convention is `StreamIdPrefix.StreamId`. An empty string means no prefix will be added to the stream IDs and names. A `null` value defaults to _ComponentID_ followed by a period.<br><br>Example: `BACnet1.DeviceId.ObjectTypeObjectId.PropertyIdentifier`<br><br>**Note:** If you change the **StreamIdPrefix** of a configured adapter (for example, when you delete and add a data source), you need to restart the adapter for the changes to apply. New streams are created on adapter restart and pre-existing streams no longer update.<br><br>Allowed value: any string<br>Default value: null|
```
[the remark this answers is not in the document]
Comment (osisoft-csleasman) [reply V1/Configuration/pi-adapter-for-bacnet-data-source-configuration.md]: The naming convention probably should say "{StreamIdPrefix}{StreamId}". @rmuolic what are your thoughts? If the goal is consistency, then all the docs need to be updated with this change.

File: V1/Configuration/pi-adapter-for-bacnet-data-source-configuration.md [hunk 2]
```diff
@@ -55,8 +55,8 @@ The following parameters are available to configure a BACnet data source:
...
+| **DefaultStreamIdPattern** | Optional | `string` | Specifies the default stream ID pattern to use. Possible parameters: {DeviceIPAddress}, {DeviceId}, {ObjectType}, {ObjectId}, and {PropertyIdentifier}.<br><br>Allowed value: any string<br>Default value: {DeviceId}.{ObjectType}{ObjectId}.{PropertyIdentifier}|
```
[the remark this answers is not in the document]
Comment (osisoft-csleasman) [reply V1/Configuration/pi-adapter-for-bacnet-data-source-configuration.md]: Again, I'll leave that decision up to @rmuolic

File: V1/Configuration/pi-adapter-for-bacnet-data-source-configuration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rmuolic) [reply V1/Configuration/pi-adapter-for-bacnet-data-source-configuration.md]: I agree with that change to {StreamIdPrefix}**.**{StreamId}. Note that I added a period between the two parameters; think that's correct? Tina and I can update the other 3 adapter docs with the change. @osisoft-csleasman

[the remark this answers is not in the document]
Comment (osisoft-csleasman) [reply V1/Configuration/pi-adapter-for-bacnet-data-source-configuration.md]: Actually there should not be a period. The only time the period is added automatically is when the value is null and "ComponentId." is added. That specific case is already documented. In all other scenarios, the period is only present if it is part of the StreamIdPrefix value.

**Review by rmuolic — APPROVED** [review]

**Review by osisoft-izinman — APPROVED** [review]
Use of backtick helps clarify. Looks great.

**Pull request merged**